danger/Gemfile

```diff
@@ -1,7 +1,10 @@
 source 'https://rubygems.org'
 
 gem "danger"
+gem "danger-suggester"
 gem "danger-rubocop"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vividmuimui/danger-sample/blob/52cb3991be8415173092296f7c5dd1efec42d2b1/danger/Gemfile#L5"></span>Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem <code>danger-rubocop</code> should appear before <code>danger-suggester</code>.</td>
    </tr>
  </tbody>
</table>